src/events/user.js

```diff
@@ -1,26 +1,103 @@
 import { EventEmitter } from 'events';
 import debug from 'debug';
 import moment from 'moment';
+import iCal from 'ical-generator';
+import * as Sentry from '@sentry/node';
 import envConfig from '../envConfig';
 import calendarEvent from '../lib/calendarEvent';
+import slackNotifications from '../lib/slackNotifications';
 
 const dlog = debug('that:api:sessions:events:user');
 const calEvent = calendarEvent(
   envConfig.calendarCredentals,
   envConfig.sharedCalendarId,
 );
 
+const baseUris = {
+  thatus: {
+    session: 'https://that.us/sessions/',
+    join: 'https://that.us/join/',
+  },
+  thatconference: {
+    session: 'https://www.thatconference.com/member/my-sessions',
+  },
+};
+
+const pmTemplates = {
+  thatconference: {
+    created: 'THATconferenceSessionCreated',
+    updated: 'THATconferenceSessionUpdated',
+  },
+  thatus: {
+    created: 'that.us-session-created',
+    updated: 'that.us-session-updated',
+    canceled: 'that.us-session-cancelled',
+  },
+};
+
+function createIcal({ session, user }) {
+  dlog('create ical for postmark email');
+  const iEvent = {
+    uid: `that-${session.id}@${user.site}`,
+    start: moment(session.startTime),
+    end: moment(session.startTime).add(session.durationInMinutes, 'minutes'),
+    summary: session.title,
+    description: session.shortDescription,
+    location: 'THAT.us',
+  };
+  let link;
+  if (user.site === 'www.thatconference.com') {
+    link = `${baseUris.thatconference.session}`;
+  } else {
+    link = `${baseUris.thatus.session}/${session.id}`;
+  }
+  iEvent.url = link;
+
+  const ical = iCal();
+  ical.prodId('//THAT Conference//THAT.us//EN');
+  ical.createEvent(iEvent);
+  const icalString = ical.toString();
+
+  return Buffer.from(icalString, 'ascii').toString('base64');
+}
+
 function userEvents(postmark) {
   const userEventEmitter = new EventEmitter();
   dlog('user event emitter created');
 
-  function sessionCreated({ user, session }) {
+  function sendSessionCreatedEmail({ user, session }) {
     dlog('new session created');
 
+    let TemplateAlias;
+    let link;
+
+    if (user.site === 'www.thatconference.com') {
+      TemplateAlias = pmTemplates.thatconference.created;
+      link = `${baseUris.thatconference.session}`;
```

Comment:
you don't need to make it a template literal. there is no template here.. just set it to the string.

Reply:
Oh, right, it was and didn't convert. I change it, it's worth having only the value. :)